test/unit/points/domain/test-ldu.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-new */
+
+const expect = require('chai').expect
+const moment = require('moment')
+const Ldu = require('../../../../app/points/domain/ldu')
+
+describe('services/domain/ldu', function () {
+  var id = 1231
+  var regionId = 3423
+  var code = '420A1F'
+  var description = 'test description'
+  var effectiveFrom = moment([2012, 4, 22])
+  var effectiveTo = moment([2015, 4, 22])
+
+  it('should construct a ldu domain object', function () {
+    var ldu = new Ldu(id, regionId, code, description, effectiveFrom, effectiveTo)
+
+    expect(ldu.id).to.equal(id)
+    expect(ldu.regionId).to.equal(regionId)
+    expect(ldu.code).to.equal(code)
+    expect(ldu.description).to.equal(description)
+    expect(ldu.effectiveFrom).to.equal(effectiveFrom)
+    expect(ldu.effectiveTo).to.equal(effectiveTo)
+  })
+
+  it('throws an error when any required property is undefined', function () {
```

Comment:
An equally important test would be ensuring that when properties that aren't required are left undefined that no error is thrown.